test/unit/examples/deviceProvisioningRequests/provisionExplicitAttrs.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
What about group provisioning with/without explicitAttrs? Is the any test covering that case?

Reply:
Group provisioning test cases are covered in **test/unit/provisioning/device-group-api-test.js** file.

Reply:
NTC